app/models/format.rb

```diff
@@ -0,0 +1,26 @@
+require 'strings'
+
+class Format < ApplicationRecord
+  include Swagger::Blocks
+
+  belongs_to :created_by, class_name: 'User'
+
+  validates :identifier, :name, :specification, :created_by, presence: true
+  validates :identifier, uniqueness: true
+
+  before_validation :set_identifier_from_name
+
+  def set_identifier_from_name
+    self.identifier = StringsHelper.code_identifier(name, padding: false)
```

Comment:
I think there's some danger to calling this.  A member could provide "General Acme Corp" and "General Acm2 Corp" and I'm guessing they'd map to the same identifier, and the member would get some uniqueness error which would be confusing.  Can we just keep the entire string they provide?  (see comment above about scoping it to the member internally)